singularity/.env.sample

```diff
@@ -0,0 +1,29 @@
+# Directory that contains PHOENIX format data 
+data=/where/you/have/PHOENIX/format/project/data
+
+# Directory to save data generated by dpdash
+state=/where/you/want/to/save/app/data
+
+# Path of the dpdash image file
+DPDASH_IMG=/path/to/dpdash.sif
```

Comment:
Just so we are clear, now the above three vars `data`, `state`, and `DPDASH_IMG` should be set in `.env` file only. However, an experienced developer can always set them in his terminal. Is that right?

Reply:
Exactly! I could add a note about this if you think it's worth doing so.

Also, since we no longer provide numbered arguments to `init.sh`, it will always pull those values from `.env` regardless of what you've set in your terminal. I could make it optional to provide them to `init.sh` just for backwards compatibility to that behavior.

Reply:
Please wait while I check if we need to run the extra mile ...

Reply:
Actually, the extra work is not needed since `init.sh` would mostly be a one-time thing.